provider/vsphere/environ.go

```diff
@@ -20,8 +20,6 @@ import (
 	"github.com/juju/juju/provider/common"
 )
 
-// Note: This provider/environment does *not* implement storage.
```

Comment:
Did you mean to remove this? It's still true - the root-disk-source setting is separate from the more general storage facilities.

Reply:
Yes I did, but I'll revert.